.github/workflows/ci.yml

```diff
@@ -221,7 +226,7 @@ jobs:
           name: ipfs-webui_${{ github.sha }}.car
 
       - name: Dry-run semantic release
-        if: github.ref != 'refs/heads/main'
+        if: github.ref != 'refs/heads/main' || github.event.inputs.kubo-version
         run: |
           git config user.name "ipfs-gui-bot"
           git config user.email "108953096+ipfs-gui-bot@users.noreply.github.com"
```

Comment:
Should we be skipping this step as well? It failed during my test run: https://github.com/ipfs/ipfs-webui/actions/runs/3135588411